actionpack/lib/action_dispatch/middleware/templates/rescues/layout.erb

```diff
@@ -131,6 +135,64 @@
     a:hover { color: #C52F24; }
     a.trace-frames.selected { color: #C52F24 }
 
+    @media (prefers-color-scheme: dark) {
+      body {
+        background-color: #222;
+        color: #ECECEC;
+      }
+
+      .details {
+        border-color: #666;
+      }
+
+      .summary {
+        border-color: #666;
+      }
+
+      .source {
+        border-color: #555;
+        background-color: #333;
+      }
+
+      .source .data {
+        background: #444;
+      }
+
+      .source .data .line_numbers {
+        background: #333;
+        border-color: #222;
+      }
+
+      .line:hover {
+        background: #666;
+      }
+
+      .line.active {
+        background-color: #977;
+      }
+
+      input[type="submit"] {
+        color: #EEE;
+        background-color: #535353;
+        border: none;
+        border-radius: 3px;
+        box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.2), 0 1px 2px rgba(0,0,0,0.15), 0 1px 1px rgba(0,0,0,0.15);
+        padding: 2px 7px;
+      }
+      input[type="submit"]:focus,
+      input[type="submit"]:hover {
```

Comment:
I don't think this focus or hover rule is necessary as there are no hover or focus styles applied to light mode buttons.